contrib/cmake/CMakeLists.txt

```diff
@@ -448,15 +468,7 @@ if (USE_THIRDPARTY_FROM_VCPKG)
             endif()
 
             IF(NOT CMAKE_HOST_APPLE AND NOT CMAKE_HOST_WIN32)
-                include(CheckCXXCompilerFlag)
-
-                check_cxx_compiler_flag(-fgomp HAS_FGOMP)
-                check_cxx_compiler_flag(-fopenmp HAS_FOPENMP)
-
-                set(Mega_PlatformSpecificLibs
-                      ${Mega_PlatformSpecificLibs}
-                      $<$<BOOL:${HAS_FGOMP}>:-fgomp>
-                      $<$<BOOL:${HAS_FOPENMP}>:-fopenmp>)
+                set(Mega_PlatformSpecificLibs ${Mega_PlatformSpecificLibs} -fopenmp -fgomp)
             ENDIF()
 
         ENDIF(USE_LIBRAW)
```

Comment:
Is this change intentional?  Since you were working specifically on Mac, and this would be a change for Linux  - and it seems like checking the compiler supports those flags would be a good idea.  thanks

Reply:
hm don't remember changing these, wonder if they snuck in during a dev pull